solr/tests/unittests/test_solr.py

```diff
@@ -41,7 +52,21 @@ def test_cleanup_solr_request(self):
         cleaned = SolrInterface.cleanup_solr_request(payload)
         self.assertNotIn('*', cleaned['fl'])
 
-
+        
+    def test_limits(self):
+        """
+        Prevent users from getting certain data
+        """
+        db.session.add(Limits(uid='9', field='full', filter='bibstem:apj'))
+        db.session.commit()
+        assert len(db.session.query(Limits).filter_by(uid='9').all()) == 1
+        
+        payload = {'fl': ['id,bibcode,title,full,bar'], 'q': '*:*'}
+        cleaned = SolrInterface.cleanup_solr_request(payload, user_id='9')
+        self.assertEqual(cleaned['fl'], u'id,bibcode,title,full')
+        self.assertEqual(cleaned['fq'], [u'bibstem:apj'])
+        
+    
```

Comment:
Can we add a two more tests.
  i. for when a user is specifying multiple `fqs` that conflict with their allowed ones for a given field
  ii. for when there are multiple entries for a single user in the database.